conf/modules.config

```diff
@@ -856,9 +856,14 @@ process {
             ].join(' ').trim()
             }
         publishDir = [
-            path: { "${params.outdir}/taxpasta/" },
+            [ path: { "${params.outdir}/taxpasta/" },
             mode: params.publish_dir_mode,
             pattern: '*.{tsv,csv,arrow,parquet,biom}'
+            ],
+            [ path: { "${params.outdir}/downstream_samplesheets/differentialabundance/"},
+            mode: params.publish_dir_mode,
```

Comment:
These should only be enabled if the `--generate_samplesheet_xxxx` parameters are used!